.github/workflows/ci.yml

```diff
@@ -3,17 +3,17 @@ name: Test and build
 on: [push, pull_request, release]
 
 jobs:
-  qa:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.9"
-      - uses: pre-commit/action@v3.0.0
+  # qa:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v4
+  #     - uses: actions/setup-python@v5
+  #       with:
+  #         python-version: "3.9"
+  #     - uses: pre-commit/action@v3.0.0
```

Comment:
I've probably asked this before, but is qa currently replaced by something, or are we waiting for packages to be fixed?

Reply:
I've no idea what the hell I was thinking here. And I merged before checking...